pkg/diagnostics/cluster/aggregated_logging/routes.go

```diff
@@ -0,0 +1,94 @@
+package aggregated_logging
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	routes "github.com/openshift/origin/pkg/route/api"
+)
+
+const routeUnaccepted = `
+An unaccepted route is most likely due to one of the following reasons:
+
+* No router has been deployed to serve the route.
+* Another route with the same host already exists.
+
+Try finding and deleting the other route by running the following:
+
+  oc get --all-namespaces routes --template='{{println}}{{range .items}}{{if eq .spec.host "%[2]s"}}{{.metadata.name}}{{println}}{{end}}{{end}}'
```

Comment:
Can I suggest the slightly more informative:

    oc get --all-namespaces routes --template='{{range .items}}{{if eq .spec.host "kibana.example.com"}}{{println .metadata.name "in" .metadata.namespace}}{{end}}{{end}}'

Has output like:

    logging-kibana in logging

Also, I think "Try finding and deleting" is a little too imperative. I don't want them doing this blindly. How about something like "If a router has been deployed, look for duplicate matching routes by running the following:"